src/mcp_grafana/tools/datasources.py

```diff
@@ -1,13 +1,30 @@
+from typing import List
 from mcp.server import FastMCP
 
+from mcp_grafana.grafana_types import Datasource
+
 from ..client import grafana_client
 
 
-async def list_datasources() -> bytes:
+async def list_datasources() -> List[Datasource]:
     """
     List datasources in the Grafana instance.
     """
-    return await grafana_client.list_datasources()
+    datasources = await grafana_client.list_datasources()
+    resp = []
+    # Only push a subset of fields to save on space.
+    for ds in datasources:
+        resp.append(
+            {
+                "id": ds.id,
+                "uid": ds.uid,
+                "name": ds.name,
+                "type": ds.type,
+                "isDefault": ds.is_default,
+            }
+        )
+
+    return resp
```

Comment:
I think this type annotation is incorrect since we're returning dicts not datasource objects? Probably doesn't matter for now but mypy will probably complain if/when we add it.

Perhaps we could add a more minimal `Datasource` type instead though, since the model might be expecting to see all the fields if it's passed the schema of the `Datasource` type.

Reply:
Yep, that is on the list of things to fix before I mark this as ready to review.